include/tvm/runtime/container.h

```diff
@@ -617,6 +636,18 @@ class Array : public ObjectRef {
     data_ = other.data_;
     return *this;
   }
+  /*!
+   * \brief Create a runtime::Array and expose it as ObjectPtr
+   * \tparam Args Type of argument list
+   * \param args Argument list
+   * \return The result ObjectPtr
+   */
+  template <typename... Args>
+  static ObjectPtr<Object> CreateObjectPtr(Args&&... args) {
```

Comment:
This function is quite subtle, let us try not to expose it directly. Define it in serialize.cc, use https://github.com/apache/incubator-tvm/blob/master/src/runtime/object_internal.h#L40 to get around the private protection(add a function to MoveObjectPtr)